content/cloud-computing/concepts/paas/paas.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'PaaS'
+Description: 'Platform as a service (PaaS): a service model that offers developers a platform to create, manage, and deploy their applications.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Cloud'
+  - 'Platform'
+  - 'Development'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Platform as a Service
+
+**Platform as a service (PaaS)** is a service model offered by cloud service providers that allows developers to develop and deploy apps without worrying about the underlying software and hardware their applications depend upon. When using a PaaS, the service provider provides and maintains the following:
+
+- Operating systems
+- Software development kits (SDK)
+- Frameworks
+- Libraries
+- APIs
+- Any other relevant software needed for application development
+
+In addition to the software, the service provider also provides the infrastructure needed for the application. PaaS includes the services provided by an infrastructure-as-a-service (IaaS) offering. Compared to the other two cloud services available - IaaS and software as a service (SaaS) - PaaS sits right in the middle, placing more responsibility on the service provider than IaaS and less responsibility than SaaS.
+
+## Advantages of PaaS
+
+Using a PaaS is often cheaper than building and maintaining the necessary software and hardware in-house. Many vendors charge based on a pay-per-use model in which a developer is billed only for the resources they use. If a developer were to design and maintain their platform instead of using a PaaS, they would risk incorrectly estimating how much of a resource they need leading them to pay for unused resources. PaaS providers handle scaling.
```

Comment:
"Many vendors charge based on a pay-per-use model in which a developer is billed only for the resources they use." I don't think this full sentence is necessary as this is something that should have been explained several times before.